mmv1/products/discoveryengine/product.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: Discoveryengine
+display_name: Discovery Engine
+versions:
+  - !ruby/object:Api::Product::Version
+    name: ga
+    base_url: https://discoveryengine.googleapis.com/v1alpha/
+  - !ruby/object:Api::Product::Version
+    name: beta
```

Comment:
Since this is just the same URL, you may only need the `ga` version specified here

Reply:
Thanks, got it. Removed beta version.
Then, IIUC, we do not need to test terraform-provider-google-beta?

Reply:
terraform-provider-google-beta is a superset of terraform-provider-google, so testing should be possible in either provider, and yield the same results. FWIW, our VCR tests in PRs use the beta provider to run.

Reply:
Thanks, I see.

Since there is only `ga` version, I should use both `ga` to generate both `google` provider and `google-beta` provider to run tests:
```
$ make provider VERSION=ga OUTPUT_PATH="$GOPATH/src/github.com/hashicorp/terraform-provider-google"
$ make provider VERSION=ga OUTPUT_PATH="$GOPATH/src/github.com/hashicorp/terraform-provider-google-beta"
```

Reply:
The second command should still use beta as the version, which controls how the provider is generated, but API urls should fall back to the GA url.

```
$ make provider VERSION=beta OUTPUT_PATH="$GOPATH/src/github.com/hashicorp/terraform-provider-google-beta"
```